draft-ietf-httpbis-priority.md

```diff
@@ -568,6 +587,22 @@ to do so is an implementation decision. For example, a server might
 pre-emptively send responses of a particular incremental type based on other
 information such as content size.
 
+A server or intermediary that recognizes a Priority HTTP Header Field in a
+request and which successfully applies that priority to its scheduling MUST
+ensure that the response contains a Priority HTTP Header Field
+with the same parameters and an applied parameter with a value of true.
```

Comment:
What if the parameter is not understood?

What if only some parameters were respected?

What if the response carries different priority information intentionally?

Reply:
If the parameter is not understood then it cannot be applied, by definition, so applied would be false. Similarly if only some parameters were respected, or if the response intentionally applied different values.

Reply:
FYI, there's a discussion of this going on in the Issues section, https://github.com/httpwg/http-extensions/issues/1274